src/main/java/net/library/converter/UserConverter.java

```diff
@@ -15,6 +16,7 @@ public static User of(final UserRequest userRequest) {
         var refinedMsisdn = Utils.deleteSpacesHyphens(userRequest.getPhoneNumber());
         user.setPhoneNumber(refinedMsisdn);
         user.setAddress(userRequest.getAddress());
+        user.setPassword(new SecurityConfig().passwordEncoder().encode(userRequest.getPassword()));
```

Comment:
Creates a new object every time. Should inject PasswordEncoder as a dependency instead